airbyte-cdk/python/airbyte_cdk/sources/file_based/config/file_based_stream_config.py

```diff
@@ -68,7 +70,7 @@ class FileBasedStreamConfig(BaseModel):
     validation_policy: Union[str, Any]
     validation_policies: Dict[str, Any]
     catalog_schema: Optional[ConfiguredAirbyteCatalog]
-    input_schema: Optional[Dict[str, Any]]
+    input_schema: Optional[Union[str, Mapping[str, Any]]]
```

Comment:
shouldn't this always get set to an Optional[Mapping] by `set_input_schema`?

Reply:
Is there benefit in making this `Mapping[str, ColumnType]` where `ColumnType` is a known string enum? This would make the UI more structured when the user is inputting these mappings, so there is no chance for error

Reply:
We don't currently have validation happening right in the UI, this is just a form field with string input, with validation happening on the back end.